output/components.sasslint.txt

```diff
@@ -4,7 +4,7 @@
 
 
 src/ObjectViewer/JSONLike/JSONLike.scss
-  65:21  warning  Hex notation should all be upper case  hex-notation
+  63:21  warning  Hex notation should all be upper case  hex-notation
```

Comment:
Could you please fix that errors while you're working on JSONLike.scss?

Reply:
Prettier is in conflict with linter. We have to choose which one is right.